src/renault_api/cli/renault_vehicle.py

```diff
@@ -315,6 +316,28 @@ async def update_lock_status(
             update_status_table(status_table, key, value, unit)
 
 
+async def update_res_state(
+    vehicle: RenaultVehicle, status_table: Dict[str, Any]
+) -> None:
+    """Update status table from get_vehicle_res_state."""
+    try:
+        if not await vehicle.supports_endpoint("res-state"):
```

Comment:
We do not have vehicles that do not support this endpoint yet, so we have to exclude this from code coverage. 
```suggestion
        if not await vehicle.supports_endpoint("res-state"):  # pragma: no cover
```